includes/core/classes/class-assets.php

```diff
@@ -321,6 +321,7 @@ protected function localize( int $post_id ): array {
 				'eventRestApi'    => $event_rest_api,
 				'loginUrl'        => $this->get_login_url( $post_id ),
 				'registrationUrl' => $this->get_registration_url( $post_id ),
+				'siteUrl'         => get_site_url(),
```

Comment:
`get_home_url` is a better choice here as it always refers to the public-facing URL for the site.